runtime-config.md

```diff
@@ -12,27 +12,28 @@ Presently there are `Prestart`, `Poststart` and `Poststop`.
 Hooks allow one to run code before/after various lifecycle events of the container.
 Hooks MUST be called in the listed order.
 The state of the container is passed to the hooks over stdin, so the hooks could get the information they need to do their work.
+All hooks execute in the host environment (e.g. the same namespace, cgroups, etc. that apply to the host process).
```

Comment:
On Wed, Oct 07, 2015 at 11:36:22AM -0700, Michael Crosby wrote:
> > +All hooks execute in the host environment (e.g. the same
> > namespace, cgroups, etc. that apply to the host process).
> 
> This is incorrect.  They are forked from the runtime process so they
> inherit from the runtime, not all host processes.

There are at least two runtime processes here.  There is a process
launched with (for example) ‘runC start’, which I've been referring to
as the “host process”.  That process forks and the child ends up in
the container, eventually running the config.json process binary.  I'd
been referring to the latter as the “container process”.  Do you
prefer alternative names for those two processes?